frontend/src/pages/CourseSelector/SearchCourse.jsx

```diff
@@ -8,7 +8,7 @@ import { useDebounce } from "../../hooks/useDebounce";
 
 export default function SearchCourse() {
   const [value, setValue] = useState("");
-  const debouncedSearchTerm = useDebounce(value, 200);
+  const debouncedSearchTerm = useDebounce(value, 500);
```

Comment:
I recently changed this to 200 to reduce the delay :).

Reply:
Ohh I see - I think when I merged dev it didn't take in the new changes for some reason, I'll go in to fix them up